internal/pkg/manifest/http.go

```diff
@@ -47,37 +52,47 @@ func (r *RoutingRuleConfigOrBool) UnmarshalYAML(value *yaml.Node) error {
 	return nil
 }
 
-// RoutingRuleConfiguration holds the path to route requests to the service.
+// RoutingRuleConfiguration holds options for application load balancer.
 type RoutingRuleConfiguration struct {
+	MainRoutingRule          ALBRoutingRule   `yaml:",inline"`
+	TargetContainerCamelCase *string          `yaml:"targetContainer"` // "targetContainerCamelCase" for backwards compatibility
+	AdditionalRoutingRules   []ALBRoutingRule `yaml:"additional_rules"`
+}
+
+// ALBRoutingRules returns main as well as additional routing rules as a list of ALBRoutingRule.
+func (cfg RoutingRuleConfiguration) ALBRoutingRules() []ALBRoutingRule {
```

Comment:
What do you think of renaming to just:
```suggestion
func (cfg RoutingRuleConfiguration) RoutingRules() []ALBRoutingRule {
```
I'm just saying that given that a listener rule is only applicable for an ALB, prefixing it with "ALB" seems redundant 💭

Reply:
Do you recommend changing just the name of the function or also changing the name of the struct?

Reply:
both would be great! sorry I should have been more explicit :lolsob: